packages/action/src/run.ts

```diff
@@ -67,12 +69,18 @@ export async function run() {
   core.info(`Commit SHA: ${commitSha}`);
 
   const token = core.getInput('github-token', { required: true });
+  if (!token) {
+    return core.info('Missing `github-token` variable');
+  }
   const checkName = core.getInput('name') || CHECK_NAME;
+  if (!checkName) {
+    return core.info('Missing `name` variable');
+  }
 
   let workspace = process.env.GITHUB_WORKSPACE;
 
   if (!workspace) {
-    return core.setFailed(
+    return core.info(
```

Comment:
I think it should fail, because `GITHUB_WORKSPACE` should be available always in GH Actions